soc/arm/cypress/common/cypress_psoc6_dt.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2020 Linaro Ltd.
+ * Copyright (c) 2020 ATL Electronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/** @file
+ * @brief Cypress PSoC-6 MCU family devicetree helper macros
+ */
+
+#ifndef _CYPRESS_PSOC6_DT_H_
+#define _CYPRESS_PSOC6_DT_H_
+
+#include <devicetree.h>
+
+/* Devicetree macros related to interrupt */
+#ifdef CONFIG_CPU_CORTEX_M0PLUS
+#define CY_PSOC6_DT_NVIC_MUX_INSTALL(n, isr, label)	    \
```

Comment:
Can you add a comment block here on what `n`, `isr`, and `label` are.

Reply:
Would also rename and maybe drop the 'NVIC_MUX' part, since as I understand it any cy6 driver should use the macros regardless.